Makefile

```diff
@@ -1,4 +1,4 @@
-DOCKERHUB_REPO := taccwma/$(shell cat ./docker_repo.var)
+DOCKERHUB_REPO := $(shell cat ./docker_repo.var)
```

Comment:
**Concern.** This change seems to be beyond a rename. Is it? _If so, do other files/repos need to change to accommodate it?_

Reply:
Add-on question - do we have an example of docker_repo.var?

Reply:
I tied in this camino update as well. Should only affect Jenkins tasks. Nobody should need to worry about it except for ProTx, who I'll coordinate with.

Reply:
an example is `taccwma/core-portal` - wanted to remove the leading `taccwma` in the case we wanted to push to another dockerhub repo in the future